invenio_stats/errors.py

```diff
@@ -71,6 +72,8 @@ def __init__(self, query_name):
         """
         super(RESTException, self).__init__()
         self.query_name = query_name
-        self.description = "Unknown statistic '{}'".format(query_name)
+        self.description = _(
+            "Unknown statistic '%(query_name)s'", query_name=query_name
+        )
```

Comment:
Question: Do you try if this style works with `gettext`, or should we use the following instead?
```suggestion
        self.description = _(
            "Unknown statistic '%(query_name)s'") % { "query_name": query_name }
```
The same question applies to the rest of the strings with `gettext`.

Reply:
and for non-english:

<img width="675" alt="image" src="https://github.com/user-attachments/assets/cf0cc17e-3749-423c-ba3c-1b9e59b55414" />